packages/gatsby/cache-dir/default-html.js

```diff
@@ -16,6 +16,11 @@ export default class HTML extends React.Component {
         </head>
         <body {...this.props.bodyAttributes}>
           {this.props.preBodyComponents}
+          <noscript key={`noscript`} id={`gatsby-noscript`}>
+            This site works best with JavaScript enabled. If this site uses
```

Comment:
Let's _just_ do this:

```
This site works best with JavaScript enabled.
```

The second note sorta indicates that we _expect_ issues and while we've seen issues, that's not what we want to convey!

Reply:
Alright! Will also update the snapshots

Reply:
Done